less/modalOverlay.less

```diff
@@ -57,26 +57,56 @@
         overflow: hidden;
       }
     }
+
+    .qrcodeOverlay {
```

Comment:
this looks somewhat off-center
![screen shot 2016-09-09 at 6 01 12 pm](https://cloud.githubusercontent.com/assets/549654/18406874/3c22cf1a-76b8-11e6-9023-aa0ca7adf990.png)